metadata.json

```diff
@@ -1,16 +1,44 @@
 {
   "name": "puppetlabs-dism",
-  "version": "1.2.0",
+  "version": "1.2.1",
   "author": "puppetlabs",
   "summary": "Provides a DISM puppet resource type on Windows.",
-  "license": "Apache License, Version 2.0",
+  "license": "Apache-2.0",
   "source": "https://github.com/puppetlabs/puppetlabs-dism",
   "project_page": "https://github.com/puppetlabs/puppetlabs-dism",
-  "issues_url": "https://tickets.puppetlabs.com/browse/MODULES",
-  "types": [
-  
+  "issues_url": "https://tickets.puppetlabs.com/browse/MODULES/component/12014",
+  "tags": [
+    "Deployment-Image-Servicing-and-Management",
+    "dism",
+    "feature",
+    "microsoft",
+    "iis",
```

Comment:
I get you can install IIS, but that is an artifact of using dism, not a feature. Am I missing something important?